en/docs/getting-started/quick-start-guide/quick-start-guide-binary.md

```diff
@@ -18,82 +18,79 @@ Make sure to install and set up all the [installation prerequisites]({{base_pat
 #### Step 1.1 - Initialize a project
 
 1.  Navigate to a preferred workspace folder using the command line. This is the location that is used to store the Microgateway project.
-2.  Run the following command to create a project named "petstore" . This will create the folder structure for the artifacts to be included.  Use the -a option to include the API definition to the project as follows.
+2.  Run the following command to create a project named "petstore" . This will create the folder structure for the artifacts to be included.  Use the `-a` option to include the API definition to the project as follows.
```

Comment:
```suggestion
2.  Run the following command to create a project named "petstore" . This will create the folder structure for the artifacts to be included. Use the `-a` option to include the API definition to the project as follows.
```